mxabierto/adela
# Feature: delete validation to temporal field on datasets and distribution models

### Changelog

* TODO: Delete validation to temporal field on distribution and dataset models

### How to test

* TODO: Update dataset and distribution wihtout temporal fields, It should change state
